phpcs.xml

```diff
@@ -13,8 +15,17 @@
 	<exclude-pattern>*/node_modules/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
 
+	<!-- Show progress, show the error codes for each message (source). -->
+	<arg value="ps"/>
```

Comment:
We already pass those flags while using PHPCS, but for pre-commit hook we also use `-n`, not sure now if this `arg` is read only if no flag is passed in the command.
Can you confirm if still works as expected for pre-commit hook?

Reply:
@claudiosanches command line flag adds to the one in the ruleset so this shouldn't be an issue and you can remove the `-ps` flags from the precommit hook.

Reply:
@claudiosanches I tested the pre-commit hook still works as expected.